### ISSUE TITLE
[🐛 Bug]: Chrome 103 regression: WebDriverError: unknown error: unexpected command response

### ISSUE THREAD

**Issue by niQo** [SeleniumHQ/docker-selenium#1613]
### What happened?

as described in this bug report, there is a regression with chrome 103: 

https://github.com/SeleniumHQ/selenium/issues/10799

### Command used to start Selenium Grid with Docker

```shell
It's not related to docker-selenium, it's a regression in chrome 103.
As explained in the reported bug: https://github.com/SeleniumHQ/selenium/issues/10799 it's working fine with chrome 102, until it's fixed in chrome, I suggest to rollback to chrome 102 in Selenium.
```

### Relevant log output

```shell
.
```

### Operating System

ubuntu

### Docker Selenium version (tag)

4.2.2-20220622, 4.3.0-20220624

**Comment by diemol** [comment]
You can always use the tag for version 102. We always advise against using `latest`. As soon as the new ChromeDriver release is out, we will do a release.

**Comment by niQo** [comment]
@diemol FYI new release of chromedriver is out including the fix for this issue [ChromeDriver 103.0.5060.134](https://chromedriver.storage.googleapis.com/index.html?path=103.0.5060.134/)
so it should be fixed on next release of docker-selenium
